readthedocs/projects/views/private.py

```diff
@@ -673,25 +677,63 @@ class DomainDelete(DomainMixin, DeleteView):
     pass
 
 
-@login_required
-def project_resync_webhook(request, project_slug):
+class IntegrationMixin(object):
+
+    """Project transaction mixin
+
+    This mixin will be used when we are listing webhook integrations on the
+    integrations page, instead of the webhook integration transactions.
     """
-    Resync a project webhook.
+
+    def get_success_url(self):
+        return reverse('projects_integrations', args=[self.get_project().slug])
+
+    def get_template_names(self):
+        if self.template_name:
+            return self.template_name
+        return 'projects/integration{0}.html'.format(self.template_name_suffix)
```

Comment:
What is the `template_name_suffix` used for? Doesn't seem like it's used anywhere.